desktopcollector/main.go

```diff
@@ -84,9 +85,17 @@ func newCommand(set otelcol.CollectorSettings) *cobra.Command {
 
 	rootCmd.Flags().IntVar(&httpPortFlag, "http", 4318, "The port number on which we listen for OTLP http payloads")
 	rootCmd.Flags().IntVar(&grpcPortFlag, "grpc", 4317, "The port number on which we listen for OTLP grpc payloads")
-	rootCmd.Flags().IntVar(&browserPortFlag, "browser", 8000, "The port number where we expose our data")
+	rootCmd.Flags().BoolVar(&openBrowserFlag, "openBrowser", true, "Open the browser automatically on launch")
+	rootCmd.Flags().IntVar(&browserPortFlag, "browserPort", 8000, "The port number where we expose our data")
 	rootCmd.Flags().StringVar(&hostFlag, "host", "localhost", "The host where we expose our all endpoints (OTLP receivers and browser)")
 	rootCmd.Flags().StringVar(&dbFlag, "db", "", "The path of your database file. Omitting this flag opens DuckDB in in-memory mode, with no data persisted to disk.")
+
+	if openBrowserFlag {
```

Comment:
I think you also need to remove the existing logic here: https://github.com/CtrlSpice/otel-desktop-viewer/blob/144834b57a6df27f85ec30d1ef0f3f7e3d788dff/desktopexporter/internal/server/server.go#L48-L56

Reply:
Handled in the next PR https://github.com/CtrlSpice/otel-desktop-viewer/pull/164/files#diff-feafe9fa0cd815e5dea580c2f6850366f43833c90b9b6c4e84462e0e806f4b4a